keylime/registrar_client.py

```diff
@@ -121,6 +121,13 @@ def getKeys(registrar_ip,registrar_port,agent_id):
             return None
 
         return response_body["results"]
+
+    except AttributeError as e :
```

Comment:
this doesn't look good as I think the httprequest.request doesn't do things consistently, which caused inconsistent handling in the caller, it should be fixed, doesn't need to be in this patch though.

Reply:
I fully expect a port in the registrar to requests. I have already moved over to requests in tenant code , will likely do registrar client next.